src/dhcp/aca_dhcp_state_handler.cpp

```diff
@@ -60,6 +62,31 @@ int Aca_Dhcp_State_Handler::update_dhcp_state_workitem(const DHCPState current_D
   stDhcpCfg.ipv6_address = current_DhcpConfiguration.ipv6_address();
   stDhcpCfg.port_host_name = current_DhcpConfiguration.port_host_name();
 
+  // handle dhcp dns entries
+  int cur_dns_index;
+  for (int i = 0; i < current_DhcpConfiguration.dns_entry_list_size() && i < DHCP_MSG_OPTS_DNS_LENGTH; i++) {
```

Comment:
@xieus @cj-chung - we need to discuss this. Currently, we have DNS information defined in both subnet.proto and dhcp.proto. Can we use use the one from the subnet level? Also, how many dns entries do we need? Just primary and secondary dns defined in subnet.proto?